src/get-url-to-download.ts

```diff
@@ -11,7 +11,12 @@ async function getURLToDownload(
   githubToken: string
 ): Promise<string> {
   const assetName = `cerbos_${version}_${runningEnvironment.os}_${runningEnvironment.architecture}.tar.gz`
-  const octokit = new Octokit({auth: githubToken})
+  const octokit = new Octokit({
+    auth: githubToken,
+    userAgent: process.env['GITHUB_REPOSITORY']
+      ? process.env['GITHUB_REPOSITORY']
+      : undefined
```

Comment:
Should we use something like `cerbos-setup-action` as the default instead of `undefined`?